utils/conversions.ts

```diff
@@ -36,23 +36,31 @@ export interface ITranslatable {
 
 export interface IReadonlyTranslatableMap {
   readonly [key: string]: ITranslatable;
+  readonly placeholder: ITranslatable;
 }
 export interface IReadonlyMap {
   readonly [key: string]: string;
+  readonly placeholder: string;
 }
 
 /**
  * A function that returns a Codec mapping an input key with a pre-created ITranslatable
  */
-export const toTWithMap = <T>(map: {
-  readonly [key: string]: T;
-}): t.Type<T, string, string> =>
+export const toTWithMap = <T>(
+  map: {
+    readonly [key: string]: T;
+  },
+  placeholder_key?: string
```

Comment:
Could we set the placeholder_key default here instead of giving it as in put in each certificate decoder?

Reply:
As the decode is now defined, no. We want the _placeholder_key_ attribute to be undefined as default. 
But I got you're point, since we always want a placehodler, there is no need to handle the _undefined_ case.
Do we remove that case?